python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -98,6 +122,25 @@ def __init__(self, list_series):
             self._series = list_series
             self._col = self._series._column
 
+        def offset_buffer(self, index):
```

Comment:
What is this doing? It's not obvious from the function name, so I think some docs would help.

Reply:
I'm going to revert this and use the accessors that @isVoid wrote in https://github.com/rapidsai/cuspatial/pull/660/files#diff-c522c9afb3364b1aed2b2589c0d0c260dbc634bc54844536b1d382cecb92bf29